app/_kong_plugins/ai-proxy-advanced/examples/semantic.yaml

```
@@ -4,7 +4,7 @@ extended_description: |
   Configure semantic load balancing with the AI Proxy Advanced plugin. To set up semantic routing, you must configure the following parameters:
 
   * `config.embeddings` to define the model to use to match the model description and the prompts.
-  * `config.vectordb` to define the vector database parameters. Only Redis is supported, so you need a Redis instance running in your environment.
+  * `config.vectordb` to define the vector database parameters. Currently, Redis and PGVector are supported.
```

```suggestion
  * `config.vectordb` to define the vector database parameters. Currently, Redis and pgvector are supported.
```
Looks like it's stylized in lowercase [based on the repo](https://github.com/pgvector/pgvector)